core/layout_api.php

```diff
@@ -899,6 +900,38 @@ function layout_options_for_sidebar( $p_menu_options, $p_active_sidebar_page ) {
 	}
 }
 
+/**
+ * Print dropdown menu item
+ * @param array  $p_menu_options menu item of group
+ * @return void
+ */
+function layout_sidebar_dropdown_menu( $p_menu_options ) {
+	echo '<li class="dropdown">' . "\n";
+   	echo '<a class="dropdown-toggle" data-toggle="dropdown" href="#">' . "\n";
+   	echo '<i class="menu-icon fa fa-plug"></i>' . "\n";
+   	echo '<span class="menu-text">Plugins</span>' . "\n";
```

Comment:
Hardcoded string - use lang_get() 
Maybe this should be a defaulted parameter ?